ANMN/common/incoming_handler.sh

```diff
@@ -60,11 +58,13 @@ handle_netcdf() {
 handle_zip() {
     local zipfile=$1; shift
 
+    local unzip_dir=`mktemp -d --tmpdir ${JOB_NAME}_XXXXX`
+    chmod 755 $unzip_dir  # make directory world readable
+
     # unzip the file
-    log_info "Unzipping '$zipfile' to $UNZIP_DIR"
-    mkdir -pv $UNZIP_DIR
-    local extracted_files=`mktemp`
-    unzip_file $zipfile $UNZIP_DIR $extracted_files || file_error "Failed to unzip"
+    log_info "Unzipping '$zipfile' to $unzip_dir"
+    local extracted_files=`mktemp --tmpdir=$unzip_dir`
+    unzip_file $zipfile $unzip_dir $extracted_files || file_error "Failed to unzip"
```

Comment:
just for the ease of understanding you could rename the variable extracted_files into something that reflect better the fact that is a manifest file